packages/extension/.env

```diff
@@ -3,7 +3,7 @@ NEXT_PUBLIC_API_URL=http://api.local.com
 NEXT_PUBLIC_SUBS_URL=ws://api.local.com/graphql
 NEXT_PUBLIC_DOMAIN=local.com
 
-NEXT_PUBLIC_WEBAPP_URL=http://webapp.local.com:5002
+NEXT_PUBLIC_WEBAPP_URL=http://webapp.local.com:5002/
```

Comment:
Is this intentional ?

Reply:
Yes. The variable originally had the slash at the end, but with the recent move we had, made this incompatible with some current implementations, but only on local env.